src/Tribe/Repository.php

```diff
@@ -874,8 +882,12 @@ public function by( $key, $value ) {
 		$this->current_filters[ $key ] = $value;
 
 		try {
+			$this->current_filter = $key;
+
 			$query_modifier = $this->modify_query( $key, $call_args );
 
+			$this->current_filter = null;
```

Comment:
I'm assuming the setting and unsetting is intentional. Can you toss a comment in place as to why?

Reply:
✅